packages/ts/tsconfig.json

```diff
@@ -1,65 +1,118 @@
 {
```

Comment:
This is the main file that most of the other tsconfigs are extending off of. There should be no changes to the configuration of this file. I updated the comments to match TS 5.3 config options and defaults since the older version was incorrect in some areas.